<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Review</class>
 <widget class="QMainWindow" name="Review">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Review</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDockWidget {font-size: 12px}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewData"/>
    <addaction name="actionSaveData"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionChangeWindowName"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewData"/>
   <addaction name="actionAddData"/>
   <addaction name="actionCopyData"/>
   <addaction name="actionNewFilter"/>
   <addaction name="actionNewRef"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockViewInputSelect">
   <property name="styleSheet">
    <string notr="true">QLabel {font-size: 13px}</string>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFeatureMask</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="dockMainTools">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable|QDockWidget::DockWidgetVerticalTitleBar</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Filters</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="wMainTools">
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockViewOptions">
   <property name="windowTitle">
    <string>View Options</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="wViewOptions"/>
  </widget>
  <widget class="QDockWidget" name="dockFilterOptions">
   <property name="styleSheet">
    <string notr="true">QLabel {font-size: 13px}</string>
   </property>
   <property name="windowTitle">
    <string>Filter Options</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="wFilterOptions"/>
  </widget>
  <action name="actionNewData">
   <property name="text">
    <string>New Data</string>
   </property>
  </action>
  <action name="actionNewFilter">
   <property name="text">
    <string>New Filter</string>
   </property>
  </action>
  <action name="actionNewRef">
   <property name="text">
    <string>New Ref</string>
   </property>
  </action>
  <action name="actionSaveData">
   <property name="text">
    <string>Save Data</string>
   </property>
  </action>
  <action name="actionAddData">
   <property name="text">
    <string>Add Data</string>
   </property>
   <property name="toolTip">
    <string>Insert data in current window</string>
   </property>
  </action>
  <action name="actionCopyData">
   <property name="text">
    <string>Copy Data</string>
   </property>
  </action>
  <action name="actionChangeWindowName">
   <property name="text">
    <string>Change Window Name</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R, F2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionNewData</sender>
   <signal>triggered()</signal>
   <receiver>Review</receiver>
   <slot>newDataWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNewFilter</sender>
   <signal>triggered()</signal>
   <receiver>Review</receiver>
   <slot>newFilterWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNewRef</sender>
   <signal>triggered()</signal>
   <receiver>Review</receiver>
   <slot>newReferenceWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveData</sender>
   <signal>triggered()</signal>
   <receiver>Review</receiver>
   <slot>saveToFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAddData</sender>
   <signal>triggered()</signal>
   <receiver>Review</receiver>
   <slot>loadFromFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCopyData</sender>
   <signal>triggered()</signal>
   <receiver>Review</receiver>
   <slot>copyWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChangeWindowName</sender>
   <signal>triggered()</signal>
   <receiver>Review</receiver>
   <slot>changeWindowName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newDataWindow()</slot>
  <slot>newFilterWindow()</slot>
  <slot>newReferenceWindow()</slot>
  <slot>saveToFile()</slot>
  <slot>loadFromFile()</slot>
  <slot>copyWindow()</slot>
  <slot>changeWindowName()</slot>
  <slot>newDatabaseInput()</slot>
 </slots>
</ui>
